x-pack/test/functional/es_archives/alerts/data.json

```diff
@@ -671,10 +671,7 @@
      "source":{
         "alert":{
            "name":"enabled 7.16.1 query rule",
-           "tags":[
-              "__internal_rule_id:064e3fed-6328-416b-bb85-c08265088f41",
-              "__internal_immutable:false"
-           ],
+           "tags":[],
```

Comment:
Initially, I didn't remove these tags, but migrations test failed.
https://buildkite.com/elastic/kibana-pull-request/builds/39883
Looks like because these tags were wiped out by my migration before test run

these tags are not required and not used for test migration this snapshot is using, so can be safely deleted